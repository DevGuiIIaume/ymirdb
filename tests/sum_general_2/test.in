set a 1
set b 2
set c b 3
set d 4
sum a
sum b
sum c
sum d
bye
