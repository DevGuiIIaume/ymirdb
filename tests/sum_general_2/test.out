> ok

> ok

> ok

> ok

> 1

> 2

> 5

> 4

> bye
